presto-druid/src/main/java/com/facebook/presto/druid/DruidBrokerPageSource.java

```diff
@@ -117,7 +117,7 @@ public Page getNextPage()
             JsonNode rootNode;
             try {
                 rootNode = OBJECT_MAPPER.readTree(data);
-                checkArgument(rootNode.isArray(), "broker Druid query should return Json Array");
+                checkArgument(rootNode.isArray(), "broker Druid query should return Json Array. detail error: "+data);
```

Comment:
nice catch.
I would suggest "broker Druid query should return Json Array:  " + data

Reply:
I would suggest only keep error message fix and float type fix in this commit, and change the commit message to: "Fix float value for druid connector"